src/components/learner-credit-management/cards/NewAssignmentModalButton.jsx

```diff
@@ -42,19 +42,34 @@ const NewAssignmentModalButton = ({ enterpriseId, course, children }) => {
   const [assignButtonState, setAssignButtonState] = useState('default');
   const [createAssignmentsErrorReason, setCreateAssignmentsErrorReason] = useState();
   const { displayToastForAssignmentAllocation } = useContext(BudgetDetailPageContext);
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+  const {
+    subsidyUuid, assignmentConfiguration, isSubsidyActive, isAssignable, catalogUuid,
+  } = subsidyAccessPolicy;
+  const sharedEnterpriseTrackEventMetadata = {
+    subsidyAccessPolicyId,
+    catalogUuid,
+    subsidyUuid,
+    isSubsidyActive,
+    isAssignable,
+    contentPriceCents: course.normalizedMetadata.contentPrice * 100,
+    contentKey: course.key,
+    courseUuid: course.uuid,
+    assignmentConfigurationUuid: assignmentConfiguration.uuid,
+    assignmentConfigurationActive: assignmentConfiguration.active,
```

Comment:
nit: if the assignment configuration isn't active, I don't believe the budget would be assignable/usable (since all the assignments correspond to the assignment configuration. I imagine this will always be true?

Reply:
Good to know that. Looking through the code I can see there is also an existing test that validates this behavior as well within `useSubsidyAccessPolicy`, so I'll remove the `assignmentConfigurationActive` field 👍🏽. Thanks for calling it out.